lib/functions/cli/cli-flash.sh

```diff
@@ -15,7 +15,7 @@ function cli_flash_pre_run() {
 }
 
 function cli_flash_run() {
-	if [[ "x${BOARD}x" != "xx" ]]; then
+	if [[ -n "$BOARD" ]]; then
```

Comment:
During development i had an edge case where this was causing me issues so fixed here as well

Reply:
this is good change to send it in separate pr. just replace `$BOARD` with `${BOARD}`:

`if [[ -n "${BOARD}" ]]; then`

Reply:
`${BOARD}` is useless curly brackets

Reply:
Submitted in https://github.com/armbian/build/pull/5713